modules/core/include/opencv2/core/hal/intrin_neon.hpp

```diff
@@ -1292,7 +1312,11 @@ inline int64 v_reduce_sum(const v_int64x2& a)
 #if CV_SIMD128_64F
 inline double v_reduce_sum(const v_float64x2& a)
 {
+#ifdef __aarch64__
```

Comment:
in `intrin_neon.hpp`, `CV_SIMD128_64F` is equivalent to `__aarch64__` so wipe out the old implementation.
The `else` part will never be referred.

Reply:
Done.